microservices/product-search/src/Controller/SearchController.php

```diff
@@ -14,11 +14,19 @@ class SearchController extends Controller
      */
     private $productSearchRepository;
 
+    /**
+     * @param Shopsys\MicroserviceProductSearch\Repository\ProductSearchRepository $productSearchRepository
```

Comment:
It's a bug `SlevomatCodingStandard\Sniffs\Namespaces\FullyQualifiedClassNameInAnnotationSniff.NonFullyQualifiedClassName`

I'd fix this manually, since it's one case and 3rd party package.